app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -395,6 +396,19 @@ def save_container_volumes_inventory(container_group, hashes, target = nil)
     store_ids_for_new_records(container_group.container_volumes, hashes, :name)
   end
 
+  def save_additional_attributes_inventory(entity, hashes, target = nil)
```

Comment:
On that note though...is custom_attributes the right place for this or should this be in advanced_settings?  It depends on the what is going to be store here in the refresh.

The difference is that custom_attributes are things that people manipulate and change, particularly through manageiq.  advanced_settings are extended metadata that is mostly for informational purposes.